libs/sdk-ui/src/base/headerMatching/HeaderPredicateFactory.ts

```diff
@@ -279,6 +281,18 @@ export function localIdentifierMatch(localIdOrMeasure: string | IMeasure): IHead
     };
 }
 
+/**
+ * Creates a new predicate that returns true for any header that belongs to either attribute or measure with the
+ * provided object reference.
+ *
+ * @public
+ */
+export function objRefMatch(objRef: ObjRef): IHeaderPredicate {
```

Comment:
Please add tests for this, similar to the other factories in this file.

Reply:
Test filled in.